cmd/cmd.go

```diff
@@ -38,6 +41,9 @@ var (
 	command string
 	version string
 	commit  string
+
+	// isSpecifiedOutputFilename is true when user specify the output file's name
+	isSpecifiedOutputFilename bool
```

Comment:
IMO, this variable doesn't have to call from another package, so it could be set to RootCommandOption instead o global variable.

Reply:
I thought that `RootCommandOption` is for flag.
(I think `isSpecifiedOutputFilename` is not for flag.)

But I think package global variable is not good either.
So, I had two plan when fix.

Preparing `isSpecifiedOutputFilename`:
1. As package global variable
Current way
2. As local variable
Call `strings.HasSuffix()` in Run() and Validate(), and store result into `isSpecifiedOutputFilename` as local variable. (although redundantly)

Reply:
Make sense.
and... I just reread code and realized that `Run()` does not need to receive `isSpecifiedOutputFilename` variable. In the first L134, `filepath.Split()` can have both file or directory as an argument, so you can remove the if block. Also, if L151 is not a file, `outFilename` is empty and can be used as a condition. 
Note: In this case, the file has to insert another variable.

Reply:
Thanks for your advice.
I fixed but I have a little problem.

`filepath.Split()` is useful.
But it needs "/" suffix to correctly split directory and filename.
So, you may feel that the process is redundant.

Of course, this is work good:)

https://github.com/Ladicle/tcardgen/pull/8/commits/42cb4f08007c431c52cb1d188792a157acbc0dfb